GNUmakefile

```diff
@@ -84,8 +84,10 @@ windows:
 	GOOS=windows GOARCH=amd64 go build -o bin/terraform-provider-alicloud.exe
 	tar czvf bin/terraform-provider-alicloud_windows-amd64.tgz bin/terraform-provider-alicloud.exe
 	rm -rf bin/terraform-provider-alicloud.exe
+	tar -xvf bin/terraform-provider-alicloud_windows-amd64.tgz && mv bin/terraform-provider-alicloud $(shell dirname `which terraform`)
 
 linux:
 	GOOS=linux GOARCH=amd64 go build -o bin/terraform-provider-alicloud
 	tar czvf bin/terraform-provider-alicloud_linux-amd64.tgz bin/terraform-provider-alicloud
-	rm -rf bin/terraform-provider-alicloud
\ No newline at end of file
+	rm -rf bin/terraform-provider-alicloud
```

Comment:
I don't suggest to update the linux and windows. You can add linuxcopy and windowscopy to fix it.

Reply:
OK